edgecast/rtld/lookups/lookups_client.go

```diff
@@ -65,18 +66,24 @@ func (a *Client) LookupsGetAwsRegions(params *LookupsGetAwsRegionsParams) (*Look
 		return nil, err
 	}
 
-	url := a.Config.BaseAPIURL.String() + "/v1.0/aws-regions"
-
-	request, err := a.Client.PrepareRequest("GET", url, results.Body, results.QueryParams, results.PathParams)
+	method, err := ecclient.ToHTTPMethod("GET")
 	if err != nil {
 		return nil, fmt.Errorf("LookupsGetAwsRegions: %v", err)
 	}
 
 	parsedResponse := &LookupsGetAwsRegionsOK{}
 
-	_, err = a.Client.SendRequest(request, parsedResponse)
-	if err != nil {
-		return nil, fmt.Errorf("LookupsGetAwsRegions: %v", err)
+	_, respErr := a.client.SubmitRequest(ecclient.SubmitRequestParams{
```

Comment:
It's best to just reuse the same error object

Reply:
Updated